.github/ISSUE_TEMPLATE/mentorship.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Mentorship request
+about: Interested in getting involved, but want some guidance? Submit a mentorship
+request!
+title: Mentorship request for <your-name>
+labels: mentorship
+assignees: ''
+
+---
+
+<!--
+If you're new to the project, welcome!
+
+In addition to filling out this mentorship request, consider joining the
+in-toto slack community and introducing yourself!
```

Comment:
Can we point to the CNCF slack workspace here?

Reply:
It is possible to set these templates for the whole organization. The sigstore community is doing this. But if we do somebody else have to set it up.

Reply:
Yes, we can include a link there. Let me have a look on it tomorrow. It's late already in germany (2:38AM) :)

Reply:
```suggestion
#in-toto channel on the CNCF slack workspace and introducing yourself!
Link to sign up: https://slack.cncf.io/
```